src/style/blocks/stars.css

```diff
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+}
+
+.stars--margin {
```

Comment:
Better do it with a mix, because this margin is the card's characteristic, not star's